docs/docs/en/guide/parameter/context.md

```diff
@@ -92,6 +92,9 @@ For example
 
 ![python_context_param](../../../../img/new_ui/dev/parameter/python_context_param.png)
 
+
+Attention: When the variable value contains the `\n` identifier, such as ` value = "hello \n world" `, value needs to be carried out in a special way. You need to use `print('${setValue(key=%s)}' % repr(value))`, otherwise the argument cannot be passed to the subsequent flow.
```

Comment:
Some humble suggestion, what do you think using:
value should remain unescaped, such as using `print('${setValue(key=%s)}' % repr(value))` ...